Python/Algorithms/aditya/stack_queue_array.py

```diff
@@ -0,0 +1,172 @@
+from abc import ABCMeta , abstractmethod
+
+class CommonFunctions(metaclass=ABCMeta) :
+    ''' A common  Base Class for FIFO and LIFO data Structures implemented using lists '''        
+    def __init__(self,value = None):
```

Comment:
Can you paste your output screenshot ?

Reply:
I have Updated the Screenshots , They are in my folder in branch .